core/src/main/java/com/scalar/db/storage/jdbc/JdbcAdmin.java

```diff
@@ -162,6 +164,8 @@ public void createNamespace(String namespace, Map<String, String> options)
       } else {
         execute(connection, "CREATE SCHEMA " + fullNamespace);
       }
+      createNamespaceTableIfNotExists(connection);
+      insertIntoNamespaceTable(connection, namespace);
```

Comment:
When creating a namespace, the namespace name is added to the `namespace` table.